DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: montagu
 Title: Interface with 'montagu'
-Version: 0.2.5
+Version: 0.2.6
```

Comment:
Can I suggest 0.3.0 given the scale of this!

Reply:
Done!